### ISSUE TITLE
NavbarDropdownButton incorrectly renders button with "btn" class

### ISSUE THREAD

**Issue by bgooren** [l0rdn1kk0n/wicket-bootstrap#495]
When the "btn" class is present, an (ugly) border is drawn around the dropdown button.

This is caused by the explicit "btn" class in https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/master/bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/button/dropdown/DropDownButton.html#L7

Since the component only appends new classes and does not remove them, the "btn" class always renders, even though NavbarDropDownButton tries to prevent this (by not adding the button behavior).

Please either remove the explicit "btn" class or update the NavbarDropDownButton so it removes the css class.

**Comment by bgooren** [comment]
FYI: workaround is to explicitly remove it in user code:
```java
dopDownButton.addToButton( new CssClassNameRemover( "btn" ) );
```

**Comment by martin-g** [comment]
This has been fixed several months ago with: https://github.com/l0rdn1kk0n/wicket-bootstrap/commit/14c903d6cae4e90bb7b34e29dc3329fb6a90facc

https://github.com/l0rdn1kk0n/wicket-bootstrap/blob/14c903d6cae4e90bb7b34e29dc3329fb6a90facc/bootstrap-core/src/main/java/de/agilecoders/wicket/core/markup/html/bootstrap/navbar/NavbarDropDownButton.java#L61

I have noticed it when adding support for Material Design theme (#442)

**Comment by bgooren** [comment]
Hi martin! Thanks, I didn't see that. It's because I trusted the project's readme, which mentions that 0.9.7 is the latest available version (when in fact you have released 0.9.8 and 0.10.0).
Perhaps it's a good idea to update the readme, so as not to confuse others.

**Comment by martin-g** [comment]
Updated!
Thanks!
